docs/Docker-image.md

```diff
@@ -0,0 +1,19 @@
+# Imagen de docker que se usará
+A la hora de escoger la imagen de docker que voy a usar voy a tener en cuenta los siguientes criterios:
+* Versión: se usará la versión más estable para que nuestro proyecto tenga el mejor soporte a pesar de perder ciertas funcionalidades que se hayan añadido en versiones nuevas (menos estables).
+* Tendrán preferencia versiones oficiales.
+* Tamaño (menor tamaño = mejor)
+
+Las imágenes de docker que son interesantes para mi proyecto son:
+* node: podría usarla por su estabilidad, versatilidad y compatibilidad con mi proyecto.
+* buster-slim: contenedor basado en debian que, como su nombre lo indica es liviana. No la voy a usar porque no se ajusta a las necesidades de mi proyecto.
+* grunt image: se plantea la posibilidad de crear mi propia imagen de docker.  
```

Comment:
No has puesto en ninguna de ellas el tamaño. Si ponéis criterios objetivos, es para que toméis la decisión en base a ellos y que pongáis en las diferentes opciones consideradas de qué forma cumplen, o no, los criterios.